ci/run-tests.ps1

```diff
@@ -19,6 +19,8 @@ function Force-ResolvePath {
     return $ExecutionContext.SessionState.Path.GetUnresolvedProviderPathFromPSPath($path)
 }
 
+. "$PSScriptRoot\common.ps1"
```

Comment:
add `$ErrorActionPreference = 'Continue'` ?

Reply:
Should not be needed here, just the cleanup is done weirdly where we try to delete things and don't care if it fails.
The Stop-Runtime thing itself should work fine.